index.src.html

```diff
@@ -414,16 +414,19 @@
           options: The options used to create a `Credential`.
         </pre>
     
-    :   <dfn method>requireUserMediation()</dfn>
-    ::  When {{CredentialsContainer/requireUserMediation()}} is called, the user agent MUST return
-        the result of executing <a abstract-op>Require user mediation</a> on the <a>current settings
+    :   <dfn method>preventSilentAccess()</dfn>
+    ::  When {{CredentialsContainer/preventSilentAccess()}} is called, the user agent MUST return
+        the result of executing <a abstract-op>Prevent Silent Access</a> on the <a>current settings
         object</a>.
   
         Note: The intent here is a signal from the origin that the user has signed out. That
         is, after a click on a "Sign out" button, the site updates the user's session info, and
-        calls `navigator.credentials.requireUserMediation()`. This toggles the <a>`requires user
-        mediation` flag</a>, meaning that credentials will not be automagically handed back to the
+        calls `navigator.credentials.preventSilentAccess()`. This toggles the <a>`prevent silent
```

Comment:
Can you change "toggles" to "sets"? I know it was here already, but it's not the right word.